qgsquick/from_qgis/plugin/editor/qgsquickvaluemap.qml

```diff
@@ -25,6 +25,7 @@ import QgsQuick 0.1 as QgsQuick
  */
 Item {
   signal valueChanged(var value, bool isNull)
+  property bool isReadOnly: readOnly
```

Comment:
What is the reason to introducing this variable?

Reply:
To be able to differentiate when feature form is editable and when not. (So that arrows have active / disabled color)


<table>
  <tr>
    <th>Viewing</th>
    <th>Editing</th>
  </tr>
  <tr>
    <td><img src="https://user-images.githubusercontent.com/22449698/109128155-8278ab00-774f-11eb-932f-d381d3fe85c7.jpg" width="300"/></td>
    <td><img src="https://user-images.githubusercontent.com/22449698/109128161-84426e80-774f-11eb-95fc-9401cda1e112.jpg"  width="300"/></td>

  </tr>

 </table>

Reply:
Edit: icons with colors are updated, see PR description. But the idea stays the same, form in viewing has icons grayed out and in edit are now green